shared_code/utils.py

```diff
@@ -41,7 +41,7 @@ def insert_services(data, client, collection):
 
 
 def check_similarity(new_service, existing_service):
-    regex = r'(st\.? |saint | inc\.?| nfp)'
+    regex = r'(\bst\.?\b|\bsaint\b|\binc\.?\b| nfp)'
```

Comment:
I don't see the test case , especially containing the values like St. or inc. which get partially matched and hence identified as duplicates

Reply:
Sorry I wasn't sure how to show the two together if there is a better way let me know.  

It is this test:
def test_check_similarity():
    assert distance('Crossroads Baptists Church', 'Crossroads Baptist Church') >= .9
    assert check_similarity('Crossroads Baptists Church', 'Crossroads Baptist Church')

which would've failed with the old code

Reply:
Add positive test cases "st paul soup kitchen"